solr/bin/solr.in.sh

```diff
@@ -203,3 +203,8 @@
 # a -Dsolr.environment property below. Valid values are prod, stage, test, dev, with an optional
 # label or color, e.g. -Dsolr.environment=test,label=Functional+test,color=brown
 #SOLR_OPTS="$SOLR_OPTS -Dsolr.environment=prod"
+
+# Runs solr in java security manager sandbox. This can protect against some attacks.
+# Runtime properties are passed to the security policy file (server/etc/security.policy)
+# This is experimental! It may not work at all with HDFS.
```

Comment:
> It may not work at all with HDFS.

This may not work with any Hadoop related usages. This includes backup/restore to HDFS compatible filesystems (https://lucene.apache.org/solr/guide/8_3/making-and-restoring-backups.html), storing indices on HDFS (https://lucene.apache.org/solr/guide/8_3/running-solr-on-hdfs.html), and Hadoop authentication plugin (https://lucene.apache.org/solr/guide/8_3/hadoop-authentication-plugin.html)

Reply:
Agreed, lets generalize the 'hdfs' here to all hadoop uses.